build/server/CMakeLists.txt

```diff
@@ -31,7 +31,8 @@ if(NOT CMAKE_BUILD_TYPE)
   set(CMAKE_BUILD_TYPE Release)
 endif()
 
-set(CMAKE_CXX_FLAGS "-Wall -Wextra -Wno-unused-parameter -Werror")
+# Need to turn unused-but-set-variable off due to Torchvision
+set(CMAKE_CXX_FLAGS "-Wall -Wextra -Wno-unused-parameter -Wno-unused-but-set-variable -Werror")
```

Comment:
Use set_target_properties in cmake file to add this flag only to the pytorch parts of the build that need it.